discovery-infra/test_infra/consts/env_defaults.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from test_infra import consts
+
+
```

Comment:
Could you try to separate these variables into logical groups?

Reply:
@YuviGold How? Separate classes? Or just semantic?